src/playcanvas-gltf.js

```diff
@@ -141,19 +141,44 @@ Object.assign(window, function () {
                 }
             } else {
                 // translation, rotation or scale
-                keyType = AnimationKeyableType.NUM;
+
+                var valuesContainTangents = sampler.interpolation === "CUBICSPLINE";
+
+                if (valuesContainTangents)
+                    numCurves /= 3;
+
+                switch (Math.round(numCurves)) {
+                    case 1: keyType = AnimationKeyableType.NUM; break;
+                    case 3: keyType = AnimationKeyableType.VEC; break;
+                    case 4: keyType = AnimationKeyableType.QUAT; break;
```

Comment:
This part affects default keyType, based on numCurves.  If path has specific name, like "translation", keyType will still be assigned by function ensureKeyType as before.